docs/fundamentals/app-host-overview.md

```diff
@@ -135,6 +135,25 @@ var apiservice = builder.AddProject<Projects.AspireApp_ApiService>("apiservice")
 
 The preceding code adds three replicas of the "apiservice" project resource to the app model. For more information, see [.NET Aspire dashboard: Resource replicas](dashboard/explore.md#resource-replicas).
 
+## Configure explicit resource start
+
+Project, executable and container resources are automatically started with your distributed application by default. A resource can be configured to wait for an explicit startup instruction with the `WithExplicitStart()` method. A resource configured with `WithExplicitStart()` is initialized with `KnownResourceStates.NotStarted`.
```

Comment:
nit: WithExplicitStart can be changed to an xref to API docs once they're available.

Reply:
Same with NotStarted

Reply:
Yes, I realize that not all potential `xref` bits were added at this point. I usually do fast-follow PRs once the API docs are live, and the xref service has indexed all the new APIs. While others do show up day one. Thanks for calling these out.